etc/config/c++.amazon.properties

```diff
@@ -1411,6 +1411,22 @@ compiler.msp430g530.semver=5.3.0
 compiler.msp430g621.exe=/opt/compiler-explorer/msp430-gcc-6.2.1.16_linux64/bin/msp430-elf-g++
 compiler.msp430g621.semver=6.2.1
 
+################################
+# CL430 (Texas Instruments MSP430 compiler)
+group.cl430.compilers=cl4302161
+group.cl430.baseName=TI CL430
+group.cl430.groupName=TI CL430
+group.cl430.instructionSet=msp430
+group.cl430.isSemVer=true
+group.cl430.supportsBinary=false
+
+compiler.cl4302161.class=cl430
+compiler.cl4302161.compilerType=cl430
+compiler.cl4302161.exe=/opt/compiler-explorer/ti-cgt-msp430_21.6.1.LTS/bin/cl430
+compiler.cl4302161.options='-I/opt/compiler-explorer/ti-cgt-msp430_21.6.1.LTS/include'
```

Comment:
Does the compiler need the single quotes? I'd be surprised if it did. We don't execute through a shell so these quotes will be passed to the compiler itself.

Reply:
OK, I'm sure they can be removed.

Reply:
I'll check later, but I'm just surprised the compiler accepted it :D

Reply:
or maybe I don't realise it but our code parses it....so much of the code I've forgotten

Reply:
I think the YAML parser gobbles it up

Reply:
Right! some horrible home-written stuff of mine, but I can believe it does some quote stuff in the executor. will check later! sorry for the delay

Reply:
Well, it absolutely works as-is! So I take that back :) Will merge (but I will remove them just for consistency's sake)